pkg/kosmosctl/rsmigrate/serviceexport.go

```diff
@@ -0,0 +1,105 @@
+package rsmigrate
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctlutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/i18n"
+	"k8s.io/kubectl/pkg/util/templates"
+	mcsv1alpha1 "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+)
+
+var exportExample = templates.Examples(i18n.T(`
+		# Export resource to control plane data storage center from control plane by default
+		kosmosctl export service foo  -n namespacefoo --kubeconfig=[control plane kubeconfig]
+		# Export resource to control plane data storage center from knode
```

Comment:
Currently,we do not support it

Reply:
modified,now only support two kinds of command:
`kosmosctl export service elasticsearch-logging-data -n foo # create elasticsearch-logging-data serviceexport in namespace foo of control plane`
`kosmosctl import service elasticsearch-logging-data -n foo --kubeconfig /tmp/master --to-knode knode-worker1 # create serviceimport elasticsearch-logging-data in knode-worker1 on the basis of service elasticsearch-logging-data in namespace foo of control plane`